.vscode/extensions.json

```diff
@@ -0,0 +1,32 @@
+{
+    "recommendations": [
+        "ambooth.git-rename",
+        "github.vscode-pull-request-github",
+        "gruntfuggly.todo-tree",
+        "mechatroner.rainbow-csv",
+        "mohsen1.prettify-json",
+        "ms-azuretools.vscode-docker",
+        "ms-python.black-formatter",
+        "ms-python.debugpy",
+        "ms-python.isort",
+        "ms-python.pylint",
+        "ms-python.python",
+        "ms-python.vscode-pylance",
+        "ms-toolsai.jupyter",
+        "ms-toolsai.jupyter-keymap",
+        "ms-toolsai.jupyter-renderers",
+        "ms-toolsai.vscode-jupyter-cell-tags",
+        "ms-toolsai.vscode-jupyter-slideshow",
+        "ms-vscode-remote.remote-containers",
+        "ms-vscode-remote.remote-ssh",
+        "ms-vscode-remote.remote-ssh-edit",
+        "ms-vscode-remote.remote-wsl",
+        "ms-vscode.cmake-tools",
+        "ms-vscode.makefile-tools",
+        "ms-vscode.powershell",
+        "ms-vsliveshare.vsliveshare",
+        "njpwerner.autodocstring",
+        "streetsidesoftware.code-spell-checker",
+        "trond-snekvik.simple-rst",
+    ]
+}
```

Comment:
I don't want to spend more time answering your message than actually modifying this list, so I will be short.

I'm  sorry to hear from you that autoDocstring is useless.
All these years, all the documentation that you have done...

In the settings.json file, we have this **magic line**:    `"autoDocstring.docstringFormat": "numpy",`

If you`re on windows, english version (and probably other languages too), try to:

1 position your cursor under a function name:
![image](https://github.com/user-attachments/assets/59899d46-45f6-4ca8-9974-1637c1f5fdb6)

2 press ctrl + shift + 2
voila! You`re going to immediately see the boilerplate for your docstring.
![image](https://github.com/user-attachments/assets/72451d20-3309-4235-a2ad-12589a256c4d)

Reply:
I already knew all that heheh. Its just that when someone does a fresh install that is not going to be set up, and we don't really warn about that. 

I still just prefer writing it out or using copilot though, but this can be a suggestion sure

Reply:
> I already knew all that heheh. Its just that when someone does a fresh install that is not going to be set up, and we don't really warn about that.
> 
> I still just prefer writing it out or using copilot though, but this can be a suggestion sure

That's the point of the `settings.json`, if you install the extension, it will already be configured to numpy formats.

What I found the most efficient is to create the boilerplate and use LLM to fill it.